charts/pmm/templates/service.yaml

```diff
@@ -17,7 +14,7 @@ metadata:
     {{- toYaml . | nindent 8 }}
   {{- end }}
 spec:
-  type: {{ $serviceType }}
+  type: {{ $serviceType | default "ClusterIP" }}
```

Comment:
I would remove this as it imply's a default that would not really happen since the service type will always be set.

Reply:
I tested with NodePort and works fine for the nginx ingress as well. not need to worry about one or the other.